vertica/setup.py

```diff
@@ -27,7 +27,7 @@ def get_dependencies():
         return f.readlines()
 
 
-CHECKS_BASE_REQ = 'datadog-checks-base>=8.2.0'
+CHECKS_BASE_REQ = 'datadog-checks-base>=11.0.0'
```

Comment:
```suggestion
CHECKS_BASE_REQ = 'datadog-checks-base>=11.11.1'
```

latest is 11.11.1 https://pypi.org/project/datadog-checks-base/#history